src/playerCommands.ts

```diff
@@ -21,9 +21,9 @@ export const commands = commandList({
 		args: [],
 		description: 'Prints information about the plugin.',
 		perm: Perm.none,
-		handler({output}){
```

Comment:
There are over a hundred formatting changes making it hard to see the actual code being contributed, please do not include unrelated changes in this PR